### PR TITLE
Added "Meet the Team" area + other bits

### PR DESCRIPTION
**Pull request by Matthew-Mace**
Added meet the team to index.html as well as creating about.html(empty) and changing some CSS

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -106,4 +110,56 @@ a{
 }
 #contact a{
     text-decoration-line: underline;
+}
```
Comment (Matthew-Mace) [inline CSS/style.css]: All this stuff is for the mtm bit

File: about.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <!-- Navigation Bar -->
+    <div class="navbar">
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="articles.html">Articles</a></li>
+            <li><a href="contact.html">Contact</a></li>
+            <li><a href="about.html">About</a></li>
+        </ul>
+    </div>
+    <h2>About us</h2>
```
Comment (SRugina) [inline about.html]: Where are the closing tags for the body tag and where is the html tag?
Also, why do you have a h1 with a class in an "a" tag? Just give your "a" tag that class & anything important in the h1 styling and get rid of the unnecessary h1 tags.

File: articles.html [hunk 3]
```diff
@@ -22,8 +22,11 @@
     <div class="navbar">
```
Comment (SRugina) [inline articles.html]: again unnecessary h1 tag in "a" tag. Also, Where is the starting html tag?

File: contact.html [hunk 4]
```diff
@@ -18,28 +18,23 @@
 <body>
     <!-- Primary Title -->
     <a href="index.html"><h1 class="title">Elephant Burp Studios</h1></a>
```
Comment (SRugina) [inline contact.html]: Again, unnecessary h1 in "a" tag. Again, where starting html tag?

File: index.html [hunk 5]
```diff
@@ -42,6 +43,17 @@
...
+        <p class="mtm" id="sebdetails">Seb is the head optimiser and debugger and an invaluable part of the team.</p>
+        <div class="mtm" id="james"><strong>James</strong></div><br>
+        <p id="jamesdetails">James is one of the programmers at Elephant Burp Studios and a key component to our team's dynamic</p>
+        <div class="mtm" id="matthew"><strong>Matthew Mace</strong></div><br>
+        <p id="matthewdetails">Matthew is the second programmer, and is also the founder of Elephant Burp Studios</p>
+    
+    </div>
     <!-- Version History -->
     <p id='versionhistory'>
         Version <span id='version'></span>. View the changelog <a href="changelog.html">here</a>
```
Comment (SRugina) [inline index.html]: Again, unnecessary h1 in "a" tag. Again, no starting html tag.

File: about.html (hunk 2 above)
Comment (Matthew-Mace) [reply about.html]: fixed it

File: articles.html (hunk 3 above)
Comment (Matthew-Mace) [reply articles.html]: ill do later

**Pull request merged**